hrms/payroll/doctype/salary_structure/salary_structure.js

```diff
@@ -96,16 +96,23 @@ frappe.ui.form.on("Salary Structure", {
 		frm.fields_dict['deductions'].grid.set_column_disp("default_amount", false);
 
 		if (frm.doc.docstatus === 1) {
-			frm.add_custom_button(__("Bulk Assign Structure"), () => {
-				frm.trigger("assign_to_employees")
-			}).addClass("btn-primary");
+			frm.add_custom_button(__("Income Tax Slab"), () => {
+				frappe.model.with_doctype("Income Tax Slab", () => {
+					const doc = frappe.model.get_new_doc("Income Tax Slab");
+					frappe.set_route("Form", "Income Tax Slab", doc.name);
+				});
+			}, __("Create"));
```

Comment:
Since the Income Tax Slab doesn't have any direct connection with the Salary Structure, we can skip this. Or keep it as the last option